docs/guides/capabilities/video/add-virtual-background.mdx

```diff
@@ -40,13 +40,37 @@ import DyteVideoBackgroundTransformer from '@dytesdk/video-background-transforme
 
 If you installed the package using a script tag, the `DyteVideoBackgroundTransformer` will be automatically available for use once the installation process is complete.
 
-To begin using the Dyte Background Transformer, you need to initialize it by calling the `init` method. Follow the steps below to initialize the transformer:
+2.x versions of DyteVideoBackgroundTransformer use their own rendering mechanism thus require you to disable the default per frame canvas rendering.
 
 ```js
-const videoBackgroundTransformer = await DyteVideoBackgroundTransformer.init();
+await meeting.self.setVideoMiddlewareGlobalConfig({ disablePerFrameCanvasRendering: true });
```

Comment:
We don't need this now do we

Reply:
2.0.1 & 2.0.2 still require this line to be added by the devs. 2.0.3 onwards, we are internally doing this.

I have kept it here for reference. Also calling this method multiple times is safe and doesn't cause any side effects.

Reply:
maybe add a note saying not required post 2.0.3?

Reply:
It is added exclusively to highlight it as a breaking change for any custom middlewares that our clients might have.

If any middleware is broken, migration could be done by referring to company branding sample.

Default middleware design: https://github.com/dyte-in/streams-middleware-samples/blob/main/video/company-branding.js
Design if per frame rendering is disabled: https://github.com/dyte-in/streams-middleware-samples/blob/main/video/middleware-with-per-frame-rendering-disabled/company-branding.js